src/internal/error/errors.go

```diff
@@ -148,11 +148,12 @@ func UnknownError(err error) *Error {
 
 // IsErr checks whether the err chain contains error matches the code
 func IsErr(err error, code string) bool {
```

Comment:
I think we should keep the original impel, as the input err can be normal, then this func could return false.